unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/Passport/WebInterfacePassportApiBridge.cs

```diff
@@ -21,6 +19,15 @@ await DCL.Environment.i.platform.serviceProviders.theGraph.QueryNftCollections(u
             return nftList;
         }
 
+        public async UniTask<Nft> QueryNftCollectionEthereum(string userId, string urn)
+        {
+            List<Nft> nftList = new List<Nft>();
```

Comment:
This is allocating memory that it's not used, it's gonna be assigned in the `.Then` anyway.